iamctl/pkg/utils/apiUtils.go

```diff
@@ -292,8 +296,10 @@ func buildRequestUrl(requestType, resourceType, resourceId string) (reqUrl strin
 	case IMPORT:
 		reqUrl = getResourceBaseUrl(resourceType) + IMPORT
 	case UPDATE:
-		if resourceType == APPLICATIONS {
+		if resourceType == APPLICATIONS || resourceType == CLAIMS {
 			reqUrl = getResourceBaseUrl(resourceType) + IMPORT
+		} else if resourceType == CLAIMS && TOOL_CONFIGS.AllowDelete {
+			reqUrl = getResourceBaseUrl(resourceType) + IMPORT + "?delete=true"
```

Comment:
Let's modify the SendUpdateRequest [1] method and add the delete option as a query param to the request.

[1] https://github.com/wso2-extensions/identity-tools-cli/blob/master/iamctl/pkg/utils/apiUtils.go#L149C36-L149C50

Reply:
Added a new function called `addQueryParams`. Modified the `SendUpdateRequest ` method to call this method an add the delete option as a query param to the request.